### DISCUSSION TITLE
Fix for CVE-2023-32233

### DISCUSSION THREAD

**Discussion by RMonesJVD** [spatie/browsershot#733] [Q&A]
As soon as the CVE becomes available, we will take the recommended mitigation actions mentioned at https://ubuntu.com/security/CVE-2023-32233.

Unfortunately, the creation of PDF files no longer functions due to this issue.

Currently, we have resolved it by following these steps outlined here: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#alternative-setup-setuid-sandbox

Additionally, we have added the following environment variable to the PHP FPM configuration: "env[CHROME_DEVEL_SANDBOX] = /usr/local/sbin/chrome-devel-sandbox."

Is this the correct approach, or is there another method you recommend in your package?

Thank you in advance.

Ronald

**Comment by osbre** [comment]
Would running Chromium in a Docker container solve this problem?
https://spatie.be/docs/browsershot/v2/miscellaneous-options/connection-to-a-remote-chrome-instance
